### PR TITLE
AJS-242 createiceserver throws warnings on ff43

### PR DESCRIPTION
**Pull request by johache**
FF43+ prints warnings when the ice candidates url in in a 'url' element.
Use 'urls' instead.
Also, printing a warning when createIceServer(s) is call, notifying the user that the functions are set to be deprecated.

### CODE REVIEW

**Comment by johache** [comment]
@letchoo that is for FF , 27 only. Was probably deprecated in their webrtc-adapter (FF 27 is pretty old).
I'm not the one who wrote this code in the first place, I would prefer to keep it as it is.

**Comment by letchoo** [comment]
@johache Ok noted. Thanks. Makes sense.

**Comment by johache** [comment]
Regarding FF < 38: this is managed by webrtc-adapter, we don't need to take care of it.
https://github.com/webrtc/adapter/blob/master/src/js/firefox/firefox_shim.js#L62

**Comment by letchoo** [comment]
@johache ok noted. Seems fine for now then

File: source/adapter.js [hunk 1]
```diff
@@ -574,25 +574,25 @@ if ( navigator.mozGetUserMedia
...
         if (webrtcDetectedVersion < 27) {
-          var turn_url_parts = url.split('?');
-          if (turn_url_parts.length === 1 ||
-            turn_url_parts[1].indexOf('transport=udp') === 0) {
+          var turnUrlParts = url.split('?');
+          if (turnUrlParts.length === 1 ||
+            turnUrlParts[1].indexOf('transport=udp') === 0) {
             iceServer = {
-              url : turn_url_parts[0],
+              urls : [turn_urlParts[0]],
```
Comment (johache) [inline source/adapter.js]: @letchoo Just saw that there was an error here, will be sending a new PR to correct that in a minute

**Pull request merged**